aroma/tests/test_integration.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import pandas as pd
+import subprocess
+from os.path import join, split
+
+import pytest
+
+
+def test_integration(skip_integration, nilearn_data):
+    if skip_integration:
+        pytest.skip('Skipping integration test')
+
+    test_path, _ = split(nilearn_data.func[0])
+
+    subprocess.run(f'aroma -o {test_path} -i {nilearn_data.func[0]} -m {nilearn_data.confounds[0]}')
```

Comment:
I believe that this might be figured out in #9. You could probably fix it by adding `aroma` as a console script like this:

https://github.com/Brainhack-Donostia/ica-aroma-org/blob/f793cbf4ca8ce1a14ea98e73574785530155178a/setup.py#L19-L22